deployment/v3/uitestrig/uitestrig/install.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+# Installs uitestrig automation
+## Usage: ./install.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+NS=uitestrig
+CHART_VERSION=12.0.1-B3
+
+echo Create $NS namespace
+kubectl create ns $NS
+
+
+function installing_uitestrig() {
+  ENV_NAME=$( kubectl -n default get cm global -o json |jq -r '.data."installation-domain"')
+
+  read -p "Please enter the time(hr) to run the cronjob every day (time: 0-23) : " time
+  if [ -z "$time" ]; then
+     echo "ERROT: Time cannot be empty; EXITING;";
+     exit 1;
+  fi
+  if ! [ $time -eq $time ] 2>/dev/null; then
+     echo "ERROR: Time $time is not a number; EXITING;";
+     exit 1;
+  fi
+  if [ $time -gt 23 ] || [ $time -lt 0 ] ; then
+     echo "ERROR: Time should be in range ( 0-23 ); EXITING;";
+     exit 1;
+  fi
+  
+  echo "Do you have public domain & valid SSL? (Y/n) "
+  echo "Y: if you have public domain & valid ssl certificate"
+  echo "n: if you don't have public domain & valid ssl certificate"
+  read -p "" flag
+  
+  if [ -z "$flag" ]; then
+    echo "'flag' was provided; EXITING;"
+    exit 1;
+  fi
+  ENABLE_INSECURE=''
+  if [ "$flag" = "n" ]; then
+    ENABLE_INSECURE='--set uitestrig.configmaps.uitestrig.ENABLE_INSECURE=true';
+  fi
+
+  echo Istio label
+  kubectl label ns $NS istio-injection=disabled --overwrite
+  helm repo update
+
+  echo Copy configmaps
+  ./copy_cm.sh
+
+  echo Copy secrets
+  ./copy_secrets.sh
+
+  DB_HOST=$( kubectl -n default get cm global -o json  |jq -r '.data."mosip-api-internal-host"' )
+  PMP_HOST=$(kubectl -n default get cm global -o json  |jq -r '.data."mosip-pmp-host"')
+  API_INTERNAL_HOST=$( kubectl -n default get cm global -o json  |jq -r '.data."mosip-api-internal-host"' )
+  USER=$( kubectl -n default get cm global -o json |jq -r '.data."mosip-api-internal-host"')
+
+
+  echo Installing uitestrig
+  helm -n $NS install uitestrig /home/techno-376/IdeaProjects/mosip-helm/charts/uitestrig/ \
```

Comment:
update the chart location from `/home/techno-376/IdeaProjects/mosip-helm/charts/uitestrig` to `mosip/uitestirg`